test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -294,12 +292,28 @@ public static IEnumerable<object[]> MessageSizesData
             }
         }
 
-        public static IEnumerable<object[]> TransportTypes =>
-            new[]
+        public static IEnumerable<object[]> TransportTypes()
+        {
+            if (WebsocketsSupported())
+            {
+                yield return new object[] { TransportType.WebSockets };
+            }
+            yield return new object[] { TransportType.ServerSentEvents };
+            yield return new object[] { TransportType.LongPolling };
+        }
+
+        private static bool WebsocketsSupported()
```

Comment:
I think this should be in Common because:
https://github.com/aspnet/SignalR/blob/ef7a3514cd2562ec5cffeadacee2e79bce34d64a/test/Microsoft.AspNetCore.SignalR.Tests/DefaultTransportFactoryTests.cs#L99
and
https://github.com/aspnet/SignalR/blob/3a1d4c5dd66cdcccbc61dc8e530784c8b79b9b1b/test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/HubConnectionTests.cs#L257